### ISSUE TITLE
Spring Boot login page issue

### ISSUE THREAD

**Issue by sshimanski** [zalando/logbook#460]
## Description

Unable to request default login form with logbook-spring-boot-starter in the dependencies. As soon I remove dependency or set **logbook.filter.enable=false** property - everything works as expected.

## Expected Behavior

I should be able to access login form with enabled LogbookFilter

## Actual Behavior

I have simple spring boot application with default authentication form. With logbook-spring-boot-starter - application starts, but I am unable to reach login form. For different web servers I can see next behaviour:
* Tomcat: continuous load
* Jetty: 500 error with "insufficient content written" message
* Undertow: empty reply from server with "Fixed length stream closed with with 1317 bytes remaining" message in logs

## Context

* logbook-spring-boot-starter version: 1.13.0
* Spring Boot version: 2.1.2.RELEASE

My security configs:
```
    @Override
    protected void configure(HttpSecurity http) throws Exception {
        http
            .csrf().disable()
            .authorizeRequests()
//          .antMatchers("/api/**").permitAll()
            .anyRequest().authenticated()
            .and().formLogin();
    }
```
If I uncomment line with permitAll - I still can not access login form, while api works as expected and I can see logged requests and responses.

**Comment by whiskeysierra** [comment]
Can you try to build the latest master and install it to your local maven repo? There were a lot of changes and maybe 2.0 already fixed that.

**Comment by sshimanski** [comment]
Hello, @whiskeysierra. I was able to access login form and even login with 2.0.0-SNAPSHOT version. But, unfortunately there is another issue:
```
java.lang.IllegalStateException: getOutputStream() has already been called for this response
	at org.apache.catalina.connector.Response.getWriter(Response.java:582)
	at org.apache.catalina.connector.ResponseFacade.getWriter(ResponseFacade.java:227)
	at javax.servlet.ServletResponseWrapper.getWriter(ServletResponseWrapper.java:109)
	at org.zalando.logbook.servlet.LocalResponse.getWriter(LocalResponse.java:102)
	at javax.servlet.ServletResponseWrapper.getWriter(ServletResponseWrapper.java:109)
	at javax.servlet.ServletResponseWrapper.getWriter(ServletResponseWrapper.java:109)
	at org.springframework.security.web.util.OnCommittedResponseWrapper.getWriter(OnCommittedResponseWrapper.java:149)
	at javax.servlet.ServletResponseWrapper.getWriter(ServletResponseWrapper.java:109)
	at org.springframework.security.web.util.OnCommittedResponseWrapper.getWriter(OnCommittedResponseWrapper.java:149)
	at javax.servlet.ServletResponseWrapper.getWriter(ServletResponseWrapper.java:109)
	at org.zalando.logbook.servlet.LocalResponse.getWriter(LocalResponse.java:102)
	at org.zalando.logbook.servlet.LogbookFilter.doFilter(LogbookFilter.java:57)
	at org.zalando.logbook.servlet.HttpFilter.doFilter(HttpFilter.java:31)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)```

**Comment by whiskeysierra** [comment]
That error has to be my white whale... (#146, #155, #153, #154, ...)

Would you be able to contribute a test case that reproduces this issue?

**Comment by sshimanski** [comment]
Hello, @whiskeysierra. You can find demo project [here](https://github.com/sshimanski/logbook-issue). Just run it with `mvn spring-boot:run` and try to GET http://localhost:8080/api

**Comment by whiskeysierra** [comment]
I was hoping for a test case within Logbook that I can keep as part of our test suite to make sure we don't introduce it again.

**Comment by whiskeysierra** [comment]
I believe I finally found a fix. See #484

**Comment by DanielFran** [comment]
@whiskeysierra Since PR has been merged, do you expect to provide version 2.0.0 soon?

**Comment by whiskeysierra** [comment]
https://github.com/zalando/logbook/releases/tag/2.0.0-RC.0
